qiskit/result/mitigation/utils.py

```diff
@@ -19,12 +19,20 @@
 import numpy as np
 
 from qiskit.exceptions import QiskitError
+from qiskit.utils.deprecation import deprecate_func
 from ..utils import marginal_counts
 from ..counts import Counts
 
 logger = logging.getLogger(__name__)
 
 
+@deprecate_func(
+    since="1.3",
+    package_name="Qiskit",
+    removal_timeline="in Qiskit 2.0",
+    additional_msg="The `qiskit.result.mitigation` module is deprecated in favor of "
+    "the https://github.com/Qiskit/qiskit-addon-mthree package.",
+)
```

Comment:
I think only the 3 classes above are actually public so these technically could just be removed. But it's good to deprecate them anyway and remove it all at once.